### ISSUE TITLE
Gracefully handle unreachable RPC server

### ISSUE THREAD

**Issue by lalexgap** [statechannels/boost#31]
Right now when nitro is enabled on `booster-http` it will panic on startup if it can't reach the nitro rpc server. We should handle this slightly more gracefully, similar to how booster-http handles waiting on the lotus/boost rpc endpoint

**Comment by lalexgap** [comment]
If we're switching to a payment proxy, this is no longer relevant.
